docs/sources/flow/reference/components/loki.source.file.md

```diff
@@ -11,6 +11,8 @@ title: loki.source.file
 Multiple `loki.source.file` components can be specified by giving them
 different labels.
 
+> **NOTE:**  `loki.source.file` does not handle file discovery. See the [File Globbing](#file-globbing) example below for usage with `local.file_match`.
```

Comment:
```suggestion
{{% admonition type="note" %}}
`loki.source.file` does not handle file discovery. You can use `local.file_match` for file discovery. Refer to the [File Globbing](#file-globbing) example for more information.
{{% /admonition %}}
```

We should use the admonition syntax for note formatting. 

I've tried a small rephrase on  the note text in an attempt to make it more clear that local.file_match` and be used for file discovery.  WDYT?

Reply:
@clayton-cornell love it!